WalletWasabi.Documentation/BackendDeployment.md

```diff
@@ -4,20 +4,23 @@ Consider updating the versions in `WalletWasabi.Helpers.Constants`. If the versi
 
 ```sh
 sudo apt-get update && cd ~/WalletWasabi && git pull && cd ~/WalletWasabi/WalletWasabi.Backend && dotnet restore && cd ~
-sudo service nginx stop
 sudo systemctl stop walletwasabi.service
-sudo killall tor
 bitcoin-cli stop
 sudo apt-get upgrade -y && sudo apt-get autoremove -y
+# in case a new kernel is available:
```

Comment:
How do we know?

Reply:
The output of `apt` will mention `linux-*` package. Additionally, `/var/run/reboot-required` may exist if there's a security-critical upgrade that needs reboot (AFAIK it's not there if the upgrade is not security-critical.)